lib/connections/limiter.go

```diff
@@ -172,6 +256,20 @@ func (lim *limiter) CommitConfiguration(from, to config.Configuration) bool {
 		}
 	}
 
+	if lim.timer != nil {
+		if !lim.timer.Stop() {
+			<-lim.timer.C
+		}
+		lim.timer = nil
+	}
+
+	if to.Options.ScheduledRatesEnabled {
+
+		waitTime := nextChange(to)
+		lim.timer = time.NewTimer(waitTime)
+		go lim.timerLoop(lim.timer, to)
+	}
+
```

Comment:
That should go to the beginning of the function and the rest of it skipped if scheduling is activated, as the limits are then set in the timer.

Reply:
Done